### ISSUE TITLE
Is it possible to use same viz_type for different plugins?

### ISSUE THREAD

**Issue by kiddick** [apache/incubator-superset#10718]
I am building my own plugin with some customization for standard pivot table plugin. So the I want to be able use and standard pivot table chart and mine customized too.
The problem is that for my plugin I need data as that requested from backend in standard plugin with `viz_type = pivot_table`.
As I understood `new PivotTableChartPlugin().configure({ key: 'pivot_table' }),` that key responsible for viz_type  mapping.
So If I am doing something like this `new PivotTableMineChartPlugin().configure({ key: 'pivot_table_mine' })`, I am getting error from backend, since there is no viz_type for my custom key.

In case `new PivotTableMineChartPlugin().configure({ key: 'pivot_table' })` standard plugin replaced with my custom.
To summarize I need use both plugins with same viz_type..

**Comment by amitNielsen** [comment]
Hey man I get it. you are doing the most amazing pivot table and you think everyone else should use it too .
But the  uniqueness of the `viz_type` param should not be an issue for you
go ahead and create a PR  similar to this https://github.com/apache-superset/superset-ui/pull/720
viz_type with `kiddick_pivot_type`

develop your amazing pivot table and let everyone know already.

this is a future feature: https://github.com/apache/incubator-superset/pull/10288 
that will allow you to load dynamic plugins by any public url
it means that nothing will block you from becoming # 1 pivot table you get just go for it 🚀 🚀 🦸

**Comment by kiddick** [comment]
Hi! Actually I want to create pivot_table plugin with some customization with my own usage. Since I need some modification and they will be very specific. So there is no reason to make it public. As I said before I need data for my plugin in same format as original plugin gets it from backend by 'viz_type' key - pivot_table. And I want to be able to use both plugins simultaneously.
